lib/icalendar/util/kv.ex

```diff
@@ -4,33 +4,49 @@ defmodule ICalendar.Util.KV do
   """
 
   @doc ~S"""
-    iex> ICalendar.Util.KV.build("foo", "bar")
+  Convert a key and value to an iCal line:
+
+    iex> ICalendar.Util.KV.build("foo", "bar", "bar")
```

Comment:
Yeah I didn't like this either. I was trying to avoid a big `case` statement in that function, however moving the `Value.to_ics` makes so much more sense :smile: